<launch>
    <include file="$(find darknet_ros)/launch/darknet_ros_nodelet.launch" />
    <include file="$(find usb_cam)/launch/usb_cam_nodelet.launch" />
    <include file="$(find whud_image_process)/launch/image_process_pid.launch" />
    <node pkg="whud_image_process" type="image_process_server.py" name="image_process_node" output="screen">
        
    </node>
    <node pkg="whud_image_process" type="image_process_speed.py" name="image_speed_control_node">
        
    </node>

</launch>
